cpu/nrf52/include/cpu_conf.h

```diff
@@ -107,9 +107,9 @@ extern "C" {
 /* The minimum block size which can be written is 4B. However, the erase
  * block is always FLASHPAGE_SIZE.
  */
-#define FLASHPAGE_RAW_BLOCKSIZE    (4U)
+#define FLASHPAGE_BLOCK_SIZE    (4U)
```

Comment:
Since you are doing a rename anyway, how about calling this `FLASHPAGE_WRITE_BLOCK_SIZE`

Reply:
Done